examples/server/utils.hpp

```diff
@@ -190,6 +191,7 @@ inline std::string format_chatml(std::vector<json> messages)
 struct llama_server_queue {
     int id = 0;
     std::mutex mutex_tasks;
+    std::atomic<bool> running;
```

Comment:
I don't think we need `atomic` here, because:
- `running` variable is never being read before the while loop in `start_loop`
- You can reuse the `mutex_task` in `terminate()` to have the atomicity

Reply:
Updated.

Reply:
You still have `#include <atomic>`.

Reply:
Oops, fixed.

Reply:
Thanks for noticing that. It’s good to merge I think